routes/channel.js

```diff
@@ -207,5 +264,42 @@ function authRequired(req, res, next) {
 	next()
 }
 
+function getGroup(timeframe) {
```

Comment:
you can rewrite this without the `let group`, just return the given object immediately